### PR TITLE
Add CMD key notice

### PR DESCRIPTION
**Pull request by Dhuls-Scratch**
Resolves [issuecomment-1050049569](https://github.com/ScratchAddons/ScratchAddons/pull/4320#issuecomment-1050049569)

### Changes
Added notice from ctrl-enter-post addon.json

### Reason for changes
So Mac users can know they use CMD

### Tests

Tested on Windows 10, Firefox 97.0.1

### CODE REVIEW

File: addons/mute-project/addon.json [hunk 1]
```diff
@@ -7,6 +7,13 @@
       "link": "https://scratch.mit.edu/users/TheColaber"
     }
   ],
+  "info": [
+    {
+      "type": "notice",
+      "text": "On macOS, you can also use the Cmd key.",
```
Comment (RedGuy12) [inline addons/mute-project/addon.json:13]: We usually use `On macOS, use the \"Cmd\" key instead of the \"Ctrl\" key.`

Comment (Dhuls-Scratch) [reply addons/mute-project/addon.json]: changed

**Review by Chiroyce1 — APPROVED** [review]

File: addons/mute-project/userscript.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ export default async function ({ addon, global, console }) {
   icon.loading = "lazy";
   icon.style.display = "none";
   const toggleMute = (e) => {
-    if (e.ctrlKey) {
+    if (e.ctrlKey || e.metaKey) {
```
Comment (WorldLanguages) [inline addons/mute-project/userscript.js:9]: Oh, this diff shouldn't be here. You should have created another branch.

Comment (Dhuls-Scratch) [reply addons/mute-project/userscript.js:9]: oops
might close this PR then

**Pull request closed without merging**